server/views/formPages/incident/evidence.html

```diff
@@ -21,7 +21,7 @@ <h1 class="govuk-heading-xl mainHeading">Evidence</h1>
           Was any evidence bagged and tagged?
         </legend>
 
-        <div class="govuk-radios" data-module="radios">
+        <div class="govuk-radios" data-module="govuk-radios">
```

Comment:
not sure why this is required.
From https://github.com/alphagov/govuk-frontend/releases
>Update and add data-module attributes
>You do not need to do anything if you're using Nunjucks macros and the initAll function.
>If you are not using Nunjucks macros, add a govuk- prefix to data-module attribute values

Does that suggest we're not using `initAll` correctly?

Reply:
We are using a mixture of HTML and Nunjucks because the design requested customised components. Because of this mix, I still had to isert the govuk- prefix